x/gov/client/testutil/deposits.go

```diff
@@ -11,8 +11,8 @@ import (
 	"github.com/cosmos/cosmos-sdk/testutil/network"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/x/gov/client/cli"
+	v1 "github.com/cosmos/cosmos-sdk/x/gov/types/v1"
```

Comment:
```suggestion
	"github.com/cosmos/cosmos-sdk/x/gov/types/v1"
```

ditto for all the other import where we don't necessarily need to explicitly name it v1

Reply:
I actually tried that, but doing that breaks the build, as import `"github.com/cosmos/cosmos-sdk/x/gov/types/v1"` will make `types.*` available, not `v1.*`

Reply:
oh I see